test/unit/test.sign.js

```diff
@@ -173,6 +173,19 @@ describe("sign", function() {
     }).catch(done);
   });
 
+  it("can configure a proxy", function() {
+    runSignCmd({
```

Comment:
Almost there. You need to return this value so that mocha will know to fail when the promise fails.